supplementary_style_guide/style_guidelines/links.adoc

```diff
@@ -20,11 +20,25 @@ For more information about <topic>, see xref:<link>[<link_text>].
 
 Follow these guidelines when linking externally:
 
-* Always include link text. Bare URLs (URLs without explanatory information) are unhelpful because they do not provide adequate context about the link target.
-* Use hyperlinks unless the URL is an example URL or is otherwise inaccessible to users. For information about using addresses in examples, see the _IBM Style_ guide.
-* By default, links are followable and crawlable. Do not use the "nofollow" link option unless absolutely necessary.
-* Avoid links to external sites when possible. External links can change or be unreliable.
-* Avoid deep links (to a specific page or image from another company) to prevent an ongoing maintenance responsibility and to prevent bypassing a website's legal information.
+* Avoid unnecessary links to external sites not owned and operated by Red Hat or IBM.
+Links to external sites can change or be unreliable.
+In addition, customers might infer that Red Hat endorses or supports the linked content, even if that is not the intent.
++
+[NOTE]
+====
+Links to upstream sites, such as GitHub, are considered to be external links.
+====
++
+* When possible, link to a top-level page and avoid deep links to a specific page or image.
+Deep links can break more frequently and can inadvertently bypass a site's legal notices.
+* Always include meaningful link text.
```

Comment:
This point (meaningful link text) should come after the point about not using bare URLs. 

We are making 2 points here:
1. Use link text. (aka "Don't use bare URLs.")
2. Use _meaningful_ link text. [that is, don't use "click here" and other such generic text]

Reply:
Thanks. That makes sense.

@bredamc @sbmetz @mportman12  - Can you all please take another look? Thank you!